### DISCUSSION TITLE
1CCD936D-21DF-AA86-179F-D9EAC36BCDC6

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#129] [General]
# NGC 1365 (Great Barred Spiral Galaxy)

The Great Barred Spiral Galaxy (also known as NGC 1365) is an SBb-type barred spiral galaxy about 58.8 million light-years away in the constellation Fornax. It is theorized that the Milky Way's shape when viewed face-on will resemble NGC 1365, in addition to having the same size

https://celestia.mobi/addon?item=1CCD936D-21DF-AA86-179F-D9EAC36BCDC6

**Comment by exocritic5457** [comment]
That's a distorted image of NGC 1300, not NGC 1365

**Reply by kmjordan122313-lgtm** [reply]
u might need to fix it.
